python-spec/src/somacore/data.py

```diff
@@ -323,17 +360,18 @@ def write(
         platform_config: Optional[options.PlatformConfig] = None,
     ) -> Self:
         """Writes an Arrow tensor to a subarray of the persistent object.
-        [lifecycle: experimental]
 
         The subarray written is defined by ``coords`` and ``values``. This will
         overwrite existing values in the array.
 
-        :param coords: A per-dimension tuple of scalars or slices
-            defining the bounds of the subarray to be written.
-            See :meth:`read` for details about indexing.
-        :param values: The values to be written to the subarray.  Must have
-            the same shape as ``coords``, and matching type to the array.
-        :return: ``self``, to enable method chaining.
+        Args:
+            coords: A per-dimension tuple of scalars or slices
+                defining the bounds of the subarray to be written.
+                See :meth:`read` for details about indexing.
+            values: The values to be written to the subarray.  Must have
+                the same shape as ``coords``, and matching type to the array.
+        Returns: ``self``, to enable method chaining.
```

Comment:
Some of the docstrings have a blank line between stanzas, some do not.  Do we need to be consistent?

I have a _very_ slight bias towards no blank line, to condense information when viewing in an IDE.

Reply:
I was doing this on a not-entirely-systematic method of “does it seem like there’s a lot of text here” as to whether to include or omit the extra line (so if there are really long segments, or ones that use multiple paragraphs, include a blank line, but otherwise not). There are situations like `SparseNDArray.write` where it is almost necessary given the contents.

Reply:
OK, works for me.